setup.py

```diff
@@ -18,7 +18,7 @@
     install_requires=[
         'eventsourcing>=1.2,<1.3',
         'django',
-        'avro-python3==1.8.1',
+        'avro-python3==1.7.7',
```

Comment:
Compatibility with the event schema registry we use. For now we are stuck on Avro v1.7 :/